commands/isitracist.js

```diff
@@ -0,0 +1,24 @@
+const config = require("../config.js");
+const default_choices = ["Yes", "No", "Maybe", "¯\\_(ツ)_/¯"];
+
+exports.run = async (client, message, args, level) => { 
+  choices = []
+  choices.push(...config.is_it_racist_choices);
+  choices.push(...default_choices);
+  const num = Math.floor(Math.random() * choices.length)
+  const msg = await message.channel.send(choices[num]);
```

Comment:
minor: the function.js provides a random func if you want to leverage it. https://github.com/AnIdiotsGuide/guidebot/blob/master/modules/functions.js

```
message.channel.send(choices.random());
```

Reply:
Oh neat. I guess that's pretty commonly used in discord bots